js/apps/account-ui/src/account-security/AccountRow.tsx

```diff
@@ -31,6 +35,7 @@ export const AccountRow = ({ account, isLinked = false }: AccountRowProps) => {
     try {
       await unLinkAccount(context, account);
       addAlert(t("unLinkSuccess"));
```

Comment:
Should this be
```suggestion
      addAlert(t("unLinkSuccess"), AlertVariant.success);
```
?

Reply:
No need, this is already the default value.